eng/tools/generator/cmd/v2/common/generation.go

```diff
@@ -27,6 +27,7 @@ type GenerateContext struct {
 	SpecReadmeFile   string
 	SpecReadmeGoFile string
 	SpecRepoURL      string
+	OriginalCommit   bool
```

Comment:
```suggestion
	UpdateSpecVersion   bool
```

You also need to change the bool logic.